dockerfiles/jupyter/README.ipynb

```diff
@@ -8,29 +8,67 @@
+    " * [Quick start tutorial for machine learning](https://docs.mlrun.org/en/stable/tutorials/01-mlrun-basics.html)\n",
+    " * [MLRun cheat sheet](https://docs.mlrun.org/en/stable/cheat-sheet.html#mlrun-setup)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddeb6a5c",
+   "metadata": {},
+   "source": [
+    "### MLRun CE default components\n",
     "\n",
-    "This Jupyter Notebook server is designed to work with the different options, it works out of the box with the **local** deployment mode.<br> \n",
-    "In order to work with remote MLRun service (Kubernetes or managed) you need to edit and save the [**mlrun.env**](./mlrun.env) file in the following way:\n",
+    "The MLRun CE (Community Edition) chart, by default, includes the following components:\n",
+    "* [MLRun](https://github.com/mlrun/mlrun)\n",
+    "  - MLRun API\n",
+    "  - MLRun UI\n",
+    "  - MLRun DB (MySQL)\n",
+    "* [Nuclio](https://github.com/nuclio/nuclio)\n",
+    "* [Jupyter](https://github.com/jupyter/notebook)\n",
+    "* [MPI Operator](https://github.com/kubeflow/mpi-operator)\n",
+    "* [MinIO](https://github.com/minio/minio/tree/master/helm/minio)\n",
+    "* [Spark Operator](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator)\n",
+    "* [Pipelines](https://github.com/kubeflow/pipelines)\n",
+    "* [Prometheus stack](https://github.com/prometheus-community/helm-charts)\n",
+    "  - Prometheus\n",
+    "  - Grafana"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7aeb1ea8",
+   "metadata": {},
+   "source": [
+    "### MLRun CE Addional components \n",
     "\n",
-    "```sh\n",
-    "# set remote MLRun service address, username and access-key\n",
-    "MLRUN_DBPATH=https://<service-address>\n",
-    "V3IO_USERNAME=<user>\n",
-    "V3IO_ACCESS_KEY=<access-key>\n",
-    "```\n",
+    "In addition to the default components, you can also enable the following components, if required:\n",
+    "\n",
+    "* MLRun Model monitoring: requires the installation of the following two applications. See more details in [Selecting the streaming and TSDB platforms](https://docs.mlrun.org/en/stable/model-monitoring/index.html#selecting-the-streaming-and-tsdb-platforms).\n",
+    "    * [TDEngine](https://taosdata.github.io/TDengine-Operator/en/2.2-tdengine-with-helm.html) (3.3.2.0) \n",
+    "    * [Kafka](https://github.com/bitnami/charts/tree/main/bitnami/kafka) (3.2.3)\n",
+    "* Real Time feature store, requires [Reddis](https://redis.io/docs/latest/operate/oss_and_stack/install/) installation as the feature store real time db (KV).\n",
+    "    * See also - [Configuring the online feature store](https://docs.mlrun.org/en/stable/install/kubernetes.html#configuring-the-online-feature-store)\n",
+    "\n",
+    "* Ingress for Nuclio functions (disabled by default) be configured in the [`values.yaml` file](https://github.com/nuclio/nuclio/blob/development/hack/k8s/helm/nuclio/values.yaml)\n",
```

Comment:
"* Ingress for Nuclio functions (disabled by default) be configured in the [values.yaml file](https://github.com/nuclio/nuclio/blob/development/hack/k8s/helm/nuclio/values.yaml)\n",